packages/@aws-cdk/aws-events/lib/rule.ts

```diff
@@ -341,15 +341,34 @@ export class Rule extends Resource implements IRule {
       out[key] = value;
     }
 
+    // rename keys in the 'detail' map
+    if (out.detail && Object.keys(out.detail).length > 0) {
+      const origdetail = out.detail;
+      const detailout: any = {};
+      for (let key of Object.keys(origdetail)) {
+        const value = (origdetail as any)[key];
+        if (key === 'repositoryName') {
```

Comment:
There shouldn't be any ECR-specific code in the Events library.

Why do the rename at all, by the way?

Reply:
Thanks for the heads up. Have reverted it back.